src/main/java/komo/fraczek/toukparking/charge/Math.java

```diff
@@ -0,0 +1,27 @@
+package komo.fraczek.toukparking.charge;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Collections;
+import java.util.function.BiFunction;
+import java.util.function.BinaryOperator;
+import java.util.function.UnaryOperator;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Math {
+
+    //     BiFunction caluculates value of given geometric series element
+    //     first term value is hard-coded.
+    static final BiFunction<Integer, Double, BigDecimal> countGeomSeriesTerm = (noOfEl, ratio) -> noOfEl == 1
```

Comment:
co znaczy `noOfEl`? Czy nie da się tego nazwać lepiej?

Reply:
Da się, mogłem to nazwać na przykład numberOfElement/Term